python/opensky_api.py

```diff
@@ -137,24 +317,31 @@ def _check_rate_limit(self, time_diff_noauth, time_diff_auth, func):
     @staticmethod
     def _check_lat(lat):
         if lat < -90 or lat > 90:
-            raise ValueError("Invalid latitude {:f}! Must be in [-90, 90]".format(lat))
+            raise ValueError("Invalid latitude {:f}! Must be in [-90, 90].".format(lat))
 
     @staticmethod
     def _check_lon(lon):
         if lon < -180 or lon > 180:
-            raise ValueError("Invalid longitude {:f}! Must be in [-180, 180]".format(lon))
+            raise ValueError(
+                "Invalid longitude {:f}! Must be in [-180, 180].".format(lon)
+            )
 
-    def get_states(self, time_secs=0, icao24=None, serials=None, bbox=()):
```

Comment:
You removed the unused parameter "serials". As it is not mentioned in the official documentation this seems to be ok.

Reply:
Yes, I removed this parameter in order to keep consistency with required parameters described in official docs.